reorderable.js

```diff
@@ -32,6 +32,15 @@
         }
       },
       itemDown: function (item, index, event) {
+        var dragDisabledForTarget = event.
+          target.
+          classList.
+          contains(this.props.disableDragClass);
+
+        if (dragDisabledForTarget) {
```

Comment:
We also need to call `event.stopPropagation()` before the `return` for this to work correctly in iOS Safari and iOS Chrome.